src/components/instructor/AssignmentsView.js

```diff
@@ -10,15 +16,133 @@ import {useLocation} from 'react-router-dom'
 // assignment id, title, dueDate and buttons to grade, edit, delete each assignment
 
 const AssignmentsView = (props) => {
+    const headers = ['Id', 'Title', 'Due Date', 'Course Id', 'Sec Id', 'Sec No', '', ''];
+    const [asgnmts, setAsgnmts] = useState([
+        //dummy data for testing
+        // { id: 1, title: "db homework 1", dueDate: "2024-02-01", courseId: "cst363", secId: 1, secNo: 8},
+        // { id: 2, title: "db homework 2", dueDate: "2024-02-15", courseId: "cst363", secId: 1, secNo: 8}
```

Comment:
I was thinking of leaving it until either Monday night or until final submission in case anything else we implement in other assignments break something and I need it again.